tests/openvino/native/quantization/test_sanity.py

```diff
@@ -38,6 +38,8 @@
 def test_compression(data_dir, tmp_path, model, dataset, ref_metrics):
     if is_windows() and model == "mobilefacedet-v1-mxnet":
         pytest.xfail("OMZ for Windows has version 1.2.0 pinned that is incompatible with Python 3.8+")
+    if model == "mobilenet-v3-small-1.0-224-tf":
```

Comment:
Open Model Zoo is used from the master https://github.com/openvinotoolkit/nncf/blob/847bbcede03989c703310e6bf056ce646568a76b/Makefile#L51 not form openvino-dev package. Is it possible to update open model zoo and remove pytest.xfail for "mobilenet-v3-small-1.0-224-tf"

Reply:
Are you sure that it would work since OMZ is part of OpenVINO, not NNCF? I'm not.
Also, for CI it wouldn't work.

Reply:
Done.